File: C/1419.c
#include <stdio.h>
#include <string.h>

int main(){
	int cnt = 0;
	char word[101];
	gets(word);
	
	for(int i = 0; i < strlen(word); i++){
		if(word[i]=='l' && word[i+1]=='o' && word[i+2]=='v' && word[i+3]=='e'){
			cnt++;
		}
	}
	
	printf("%d", cnt);
	
	return 0;
}

File: Java/1419.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
    	Scanner sc = new Scanner(System.in);
    	
    	String word = sc.nextLine();
    	
    	int cnt = 0;
    	
    	for(int i = 0; i < word.length(); i++){
    		if(word.charAt(i)=='l' && word.charAt(i+1)=='o' && word.charAt(i+2)=='v' && word.charAt(i+3)=='e'){
    			cnt++;
    		}
    	}
    	
    	System.out.println(cnt);
    }
}